src/config.js

```diff
@@ -57,9 +57,9 @@ function getConfig() {
       cache_time: process.env.CACHETIME
         ? process.env.CACHETIME
         : data.env_variables.CACHETIME,
-      debug: process.env.DEBUGLOG
-        ? process.env.DEBUGLOG
-        : data.env_variables.DEBUGLOG,
+      debug: () => process.env.DEBUG
```

Comment:
```suggestion
      debug: process.env.DEBUG
```
Keeping this as just a variable along with everything else can reduce the overhead of internal knowledge of the config file. To allow it to be used uniformly.

Reply:
@confused-Techie this was made to actually make the logger component testable, because having everything loaded at the startup makes it impossible to test. I'm not sure what you mean with "overhead of internal knowledge" but untestable code is not good as a general concept.

Reply:
You do have a point that having it loaded once at startup makes it hard to test. 

But to explain "overhead of internal knowledge": 
My concern is that everything else in `config.js` is exported as a variable. So another contributor would expect to be able to use `const { variable } = require("config.js").getConfig();`. As that's what's used everywhere else, but with this change any other contributor would have to know that `debug` is the only thing they can't import this way. They have to know explicitly the one exception, instead of being able to infer it's value from everything else. Which while not bad, does raise the barrier to entry on the codebase.

What if instead we don't import `logger` at the top of the file. We could import `logger` within the tests themselves. So that way we set an env variable before requiring the module. Not sure if Node's module caching would cause any issues but would make this section testable without having to make a special use case